repos/system_upgrade/common/actors/scandasd/libraries/scandasd.py

```diff
@@ -14,7 +14,7 @@ def process():
         # the file has to be copied into the targetuserspace container first,
         # then it can be included into the initramfs ==> both messages are
         # needed to be produced
-        api.produce(TargetUserSpaceUpgradeTasks(copy_files=[CopyFile(src=DASD_CONF)]))
+        api.produce(TargetUserSpaceUpgradeTasks(copy_files=[CopyFile(src=DASD_CONF)], install_rpms=['s390utils-core']))
```

Comment:
Even if the system boots from a DASD disk the `DASD_CONF` doesn't have to exist. Would it make more sense to always install the `s390utils-core`?

Reply:
that's what this does

Reply:
oh now I see what you mean

Reply:
@pirat89 that's a good point, no?

Reply:
good catch!